application/src/main/java/org/thingsboard/server/service/entitiy/DefaultTbNotificationEntityService.java

```diff
@@ -36,9 +36,13 @@
 import org.thingsboard.server.common.data.id.DeviceId;
 import org.thingsboard.server.common.data.id.EdgeId;
 import org.thingsboard.server.common.data.id.EntityId;
+import org.thingsboard.server.common.data.id.RuleChainId;
```

Comment:
in general, **notifyEntity** method has only single line
`logEntityAction(tenantId, entityId, entity, customerId, actionType, user, e, additionalInfo);`

Does it really necessary to move log entity action into notifying entity method?

Reply:
![image](https://user-images.githubusercontent.com/3466101/170943213-21e3ea92-8575-457d-8543-5d12d13ab252.png)

Reply:
Ok

Reply:
these changes are not approved by @ashvayka @YevhenBondarenko  
cancelling them